types/pg-cursor/pg-cursor-tests.ts

```diff
@@ -47,9 +47,12 @@ const handleFn: ResultCallback<string> = (err: Error | undefined, rows: string[]
 handle.read(100, handleFn);
 
 // Returns promise when no callback
-handle.read(100).then(() => {
-    // Finished
-});
+handle
```

Comment:
To test the type more thorough you could type the cursor here with some arbitrary type with a property and the check that the returned rows are indeed properly typed:

```const cursor = new Cursor<{foo: string}>('SELECT * FROM table');
const handle = client.query(cursor);
cursor.read((rows) => {
rows.foo
})